OpenQA/Isotovideo/Utils.pm

```diff
@@ -64,4 +65,27 @@ sub checkout_git_repo_and_branch {
     $bmwqemu::vars{$dir} = File::Spec->rel2abs($local_path);
 }
 
+=head2
```

Comment:
I sort of wish there was a better namespace to put this routine... But I don't wanna say bmwqemu...

Reply:
I needed to put it in testapi. See comment below:
>  I needed to move compat_args() to testapi.pm as I had cycling include problems with Utils.pm.